doc/howtos/bluetoothspeaker.rst

```diff
@@ -0,0 +1,141 @@
+Bluetooth Speaker
+#################
+
+How to setup your intel-iot-refkit image to function as a bluetooth speaker
+===========================================================================
+
+For configuring the bluetooth speaker setup you should use the
```

Comment:
I think it would be easier if I come back and do some grammar editing after the technical issues are updated. (Unless you'd like me to comment on that now.)